app/packs/controllers/dropzone_controller.js

```diff
@@ -75,10 +87,14 @@ export default class extends Controller {
 
   bindEvents() {
     this.dropZone.on("addedfile", file => {
+      if (this.checkForDuplicates(file.name) === true) {
```

Comment:
Do we need to compare here? Isn't this method returning a boolean?

Reply:
Updated in latest commit.